src/test/java/com/google/api/codegen/testdata/discoveries/py/py_logging.v2beta1.json.baseline

```diff
@@ -27,18 +30,19 @@ credentials = GoogleCredentials.get_application_default()
 service = discovery.build('logging', 'v2beta1', credentials=credentials)
 
 
-
 list_log_entries_request_body = {
-# TODO: Add desired entries of the 'list_log_entries_request_body' dict
+    # TODO: Add the desired entries.
 }
 
+
 entries = service.entries()
 request = entries.list(body=list_log_entries_request_body)
 while request is not None:
     response = request.execute()
 
     for log_entry in response['entries']:
-        # TODO: Add code here to process each 'log_entry' resource
+        # TODO: Use log_entry:
+        pprint(log_entry)
```

Comment:
I don't think there was any UX testing, at some point when we were talking about consistency we thought printing outputs might be useful. If that's something we want to back off from right now, I'm fine with removing it.